nntrainer/tensor/manager.h

```diff
@@ -30,6 +30,7 @@
 #include <vector>
 
 #include <basic_planner.h>
+#include <cache_loader.h>
```

Comment:
cache_loader.h is not included in this PR.

Reply:
thanks. it removed.